.travis.yml

```diff
@@ -2,4 +2,5 @@ language: node_js
 node_js:
   - "8.11.3"
 script: 
-  - npm run size
\ No newline at end of file
+  - npm run size
```

Comment:
reversed order of those scripts makes more sense to me, would prefer running test first and then size